src/animate.py

```diff
@@ -24,28 +25,61 @@
 matplotlib.rc('text', usetex='false')
 
 
-def run(country, to_plot='confirmed', save=False, path=None):
+def run(country, region= "", to_plot='confirmed', save=False, path=None,cached=False,repeat=True):
```

Comment:
Mi faresti pep8 formatting per favore? (e.g. spazi tra gli argomenti delle funzioni) 

for readability :)